js/main.js

```diff
@@ -0,0 +1,43 @@
+let x = min(333, 55);
+function min(a, b) {
+  if (a < b) {
+    return a;
+  } else {
+    return b;
+  }
+}
+console.log(x);
+
+let y = max(333, 55);
+function max(a, b) {
+  if (a > b) {
+    return a;
+  } else {
+    return b;
+  }
+}
+console.log(y);
+
+// function palindrome(str){
+//     let str2 = str.split('');
+//     str2 = str2.reverse();
+//     str2 = str2.join('');
+//     if (str == str2) {
+//         return true;
+//     }
+//         else return false;
+//     {
+//     }
+// }
+// console.log(palindrome(''))
```

Comment:
Why did you keep this code?

Reply:
I have kept for myself the full version of the abbreviated form

Reply:
Just remember leaving something "for myself" is unacceptable on real projects